statsmodels/regression/linear_model.py

```diff
@@ -635,7 +635,9 @@ class WLS(RegressionModel):
     weights : array
         The stored weights supplied as an argument.
 
-    See regression.GLS
+    See also
```

Comment:
Is "also" supposed to be capitalized?  I've seen it both ways

Reply:
Yes should be.  Was standardized in the past couple fo weeks.

Reply:
Done